pom.xml

```diff
@@ -54,7 +55,14 @@
 
   <properties>
     <version.org.hawkular.bus>0.7.0.Final</version.org.hawkular.bus>
+    <version.org.hawkular.accounts>1.0.14.Final</version.org.hawkular.accounts>
     <api-check-activated>true</api-check-activated>
+
+    <!-- integration tests distribution properties -->
+    <test-dist-version.org.hawkular.bus>${version.org.hawkular.bus}</test-dist-version.org.hawkular.bus>
+    <test-dist-version.org.hawkular.accounts>${version.org.hawkular.accounts}</test-dist-version.org.hawkular.accounts>
+    <test-dist-version.org.hawkular.commons>0.2.1.Final</test-dist-version.org.hawkular.commons> <!-- C* -->
+    <test-dist-version.org.keycloak>1.3.1.Final</test-dist-version.org.keycloak>
```

Comment:
Is it ok like this? I think is better to be able to change version of accounts, bus for test distribution.